### ISSUE TITLE
Spin Auth

### ISSUE THREAD

**Issue by davem2020** [ubahnverleih/WoBike#110]
I am getting an empty response when trying to obtain auth token from Spin via https://web.spin.pm/api/v1/auth_tokens.  This had previously been working for me. Anybody else having this issue?

**Comment by ChristieDesnoyer30** [comment]
Yes it seems like they must have changed something! I previously had been able to access it, now no bueno.
